src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/utils/urlHelper.ts

```diff
@@ -75,18 +76,23 @@ export function getUpgradeAuthLevelUrl(reqAuthLevel: string) {
   return `https://${getHostname()}/ui/authentication/upgrade?goTo=${encodeURIComponent(redirect)}&reqAuthLevel=${reqAuthLevel}`;
 }
 
-export const getEnvironmentLoginUrl: () => string = () => {
+export const getEnvironmentLoginUrl = (oidcprovider: string) => {
```

Comment:
It would be great to add some tests to this function, to validate the different outcomes based on the ifs and the throw statement.

- [ ] Add tests

Reply:
Done